### PR TITLE
Hcal rel val update9 xy v2

### PR DESCRIPTION
**Pull request by bcmcms**
Fixed issues with occupancy vs ieta graphs not being filled, and added needed error bars.

### CODE REVIEW

File: DQMOffline/Hcal/src/CaloTowersDQMClient.cc [hunk 1]
```diff
@@ -132,10 +145,14 @@ int CaloTowersDQMClient::CaloTowersEndjob(const std::vector<MonitorElement*> &hc
          else
             phi_factor = 36.;
        }
-       if(ieta >= 0) ieta -= 1; // -41 -1, 0 40  - to bring back to histo num
 
-       cnorm = sumphi / phi_factor;
-       occupancy_vs_ieta->Fill(double(ieta), cnorm);
+      std::cout << "sumphi = " << sumphi << std::endl;
```
Comment (dmitrijus) [inline DQMOffline/Hcal/src/CaloTowersDQMClient.cc]: Please remove this, std::cout is not allowed.

Comment (bcmcms) [reply DQMOffline/Hcal/src/CaloTowersDQMClient.cc]: Sorry about that, it's fixed now.

**Comment by dmitrijus** [comment]
+1

File: DQMOffline/Hcal/src/CaloTowersDQMClient.cc [hunk 2]
```diff
@@ -78,18 +78,22 @@ int CaloTowersDQMClient::CaloTowersEndjob(const std::vector<MonitorElement*> &hc
...
    float cont;
+   float conte;
    float fev = float(nevent);
 
    for (int i = 1; i <= nx; i++) {
-      cont = Ntowers_vs_ieta -> getBinContent(i) / fev ;
+      cont  = Ntowers_vs_ieta -> getBinContent(i) / fev ;
+      conte = pow(Ntowers_vs_ieta -> getBinContent(i),0.5) / fev ;
       Ntowers_vs_ieta -> setBinContent(i,cont);
+      Ntowers_vs_ieta -> setBinError(i,conte);
```
Comment (davidlange6) [inline DQMOffline/Hcal/src/CaloTowersDQMClient.cc]: should you not include the error on fev?

Comment (hatakeyamak) [reply DQMOffline/Hcal/src/CaloTowersDQMClient.cc]: I am not sure if we want to do that, as ntowers and fev are strongly correlated. So, we cannot add two errors quadratically. The relval stats is typically 10k, so the error on fev is 1% which is typically negligible.

**Comment by davidlange6** [comment]
+1

**Pull request merged**